ink/goddess/reward_joi

```diff
@@ -779,20 +783,20 @@ Nod your head when you are finished.
 = cum_comments
     {shuffle:
     - Such a big {player_name}!
-    - Watching my slaves cum always gets me so fucking wet.
+    - The scene of a slaves ejaculation always warms my nether regions.
     - Good job little one.
-    - What a big load from such a small cock.
-    - Mmmm... impressive
-    - I'm almost tempted to go over there and clean you up myself... Maybe one day.
+    - Such an impreesive load from one so puny.
+    - Mmmm... impressive.
+    - I'm almost tempted to go over there and clean you up myself... Perhaps one day.
 }
 
 ->->
 
 = cei_instructions
 {shuffle:
-   - Go get a small cup to fill when I order you to cum.
-   - I want you to shoot that load into your hand and lick it all up when I order you to cum.
-   - Now go get a spoon to fill with your hot load when I tell you to cum.
+   - Fetch a small vessel. You will fill it when I order you to cum.
+   - I want you to shoot your load into the palm of your hand and lick it all up after I order you to cum.
```

Comment:
I think the word 'shoot' isn't quite like her... so perhaps it would read better as "I want you to spew your seed into your hand and lick it all up after I order you to cum"

You could switch to 'the palm of...', if you want. I don't think you have to.

Reply:
I agree shoot doesn't sound like her. Great catch :)